tracks/csharp/exercises/resistor-color/mentoring.md

```diff
@@ -0,0 +1,97 @@
+### Reasonable solutions
+#### Array Based
+``` 
+public static class ResistorColor
+{
+    private static readonly string[] resistorColors = { "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white" };
+
+    public static int ColorCode(string color) => System.Array.IndexOf(resistorColors, color);
+
+    public static string[] Colors() => resistorColors;
+}
+```
+#### Dictionary Based
+Based on Horatiu's solution
+```
+﻿using System.Collections.Generic;
+ using System.Collections.ObjectModel;
```

Comment:
Tiny nit: unneeded space at the beginning of the line

Reply:
Some weird tick in Rider.